editor.planx.uk/src/pages/FlowEditor/components/Team/MembersTable.tsx

```diff
@@ -79,6 +81,19 @@ export const MembersTable: React.FC<{ members: TeamMember[] }> = ({
               <TableCell>{member.email}</TableCell>
             </StyledTableRow>
           ))}
+          {showAddMemberButton && (
+            <StyledTableRow>
+              <TableCell>
+                <AddButton
+                  onClick={() =>
+                    alert("Sorry! This feature is not complete yet...")
+                  }
```

Comment:
I know this is still draft (looks like great start!) - but just wanted to flag that it might be useful to checkout `src/lib/featureFlags.ts` here - a small resource to help facilitate iterative PRs like this one :slightly_smiling_face: 

You could: 
- Add a new value to `AVAILABLE_FEATURE_FLAGS` array
- Add a condition to or replace `showAddMemberButton` with `hasFeatureFlag("{your new value}")`
- Run `window.featureFlags.toggle("{your new value")` in the browser console to turn on/off

Reply:
& an old example of how to then reference/use `toggleFeatureFlag` in tests ! https://github.com/theopensystemslab/planx-new/pull/1563/files